docs/requirements/apod/020-controlplane-layer3-management.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Documentation     Controlplane VM need connectivity a management VRF.
+...
+...               Management network is layer3, so it also needs to have a gateway.
+...               
+...               Management network requires internet connectivity.
+...               
+...               Management network requires connectivity to other SAP networks and all CC+1 regions.
+...               
+...               Controlplane traffic depends on Path MTU Discovery. Upstream device must generate 
+...               an ICMP Frag-needed packet if Controlplane VM sends packet that is bigger than 
+...               interface MTU of the upstream device. 
+...               
+Library           SSHLibrary
+Default Tags      aPod      Controlplane
+
+*** Variables ***
+${HOST}            minion0.cc.qa-de-1.cloud.sap
+${INTERNET_IP}     8.8.8.8
+${CORP_IP}         <github-wdf-ip>
+${USERNAME}        test
+${PASSWORD}        test
+
+*** Test Cases ***
+
+Management network internet connectivity
+    Skip    Given Open Connection controlplane VM And Log In
+    When ${output} = Execute Command "ping ${INTERNET_IP}"
+    Then Should Contain ${output} "64 bytes from ${INTERNET_IP}"
+
+Management network SAP corp connectivity
+    Skip    Given Open Connection controlplane VM And Log In
+    When ${output} = Execute Command "ping ${CORP_IP}"
+    Then Should Contain ${output} "64 bytes from ${CORP_IP}"
+
```

Comment:
Aren't we missing the case for _fragmentation needed_ packets? I could imagine sending an oversized ping to a destination on DMVPN, e.g. ap-jp-2 and then check the PMTUD cache.

Reply:
we are missing lots of test cases tbh we just wanted to propose this kind of dokumentation to be able to add test cases later.